aliases/available/general.aliases.bash

```diff
@@ -55,6 +56,7 @@ alias cd..='cd ..'       # Common misspelling for going up one directory
 alias ...='cd ../..'     # Go up two directories
 alias ....='cd ../../..' # Go up three directories
 alias -- -='cd -'        # Go back
+alias d='cd /home/$USER/Downloads' # Go to the Downloads directory
```

Comment:
This is a bit too short and already taken by a function for the fasd plugin. please rename to something that won't clobber or move to your private alias file.